src/bitcoin/bitcoinCoreRpc.ts

```diff
@@ -2,7 +2,12 @@
 import Client from "bitcoin-core";
 import { Transaction } from "bitcoinjs-lib";
 import { getLogger } from "log4js";
-import { BitcoinConfig } from "../config";
+import {
+  BitcoindCoreRpcBasicAuth,
```

Comment:
🔧 I find the names a bit confusing are they all seem to be at the same level whereas actually, 2 are for Authentication and one config.

Reply:
Feel free to propose different names in a follow-up PR :)